app/controllers/catalog_controller.rb

```diff
@@ -133,24 +131,24 @@ class CatalogController < ApplicationController
     # solr fields to be displayed in the show (single result) view
     #   The ordering of the field names is the order of the display
     config.add_show_field 'title_latin_display_ssm', label: 'Title'
-    config.add_show_field 'author_person_display_ssm', label: 'Author', link_to_facet: :all_authors_facet_ssim
-    config.add_show_field 'author_corp_display_ssm', label: 'Corporate Author', link_to_facet: :all_authors_facet_ssim
-    config.add_show_field 'author_meeting_display_ssm', label: 'Conference Author', link_to_facet: :all_authors_facet_ssim
+    config.add_show_field 'author_person_display_ssm', label: 'Author', link_to_facet: :all_authors_facet
+    config.add_show_field 'author_corp_display_ssm', label: 'Corporate Author', link_to_facet: :all_authors_facet
+    config.add_show_field 'author_meeting_display_ssm', label: 'Conference Author', link_to_facet: :all_authors_facet
     config.add_show_field 'uniform_title_display_ssm', label: 'Uniform Title'
     config.add_show_field 'additional_title_display_ssm', label: 'Additional Titles'
     config.add_show_field 'format', label: 'Format'
     config.add_show_field 'overall_imprint_display_ssm', label: 'Published'
     config.add_show_field 'copyright_display_ssm', label: 'Copyright Date'
     config.add_show_field 'edition_display_ssm', label: 'Edition'
     config.add_show_field 'phys_desc_ssm', label: 'Physical Description'
-    config.add_show_field 'language_facet_ssim', label: 'Language'
+    config.add_show_field 'language_ssim', label: 'Language'
```

Comment:
@cdmo should this be `language_facet`?

Reply:
No because we display language in records and we also include it in our index as part of search, so, I split the field into two: one for faceting and one for indexing/displaying. See catalog controller.

Reply:
I see 👍  thanks!